src/adagrams.js

```diff
@@ -1,8 +1,114 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
+const letterPool = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4, 
+  E: 12, 
+  F: 2,
+  G: 3, 
+  H: 2,
+  I: 9, 
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1 
 };
 
+class Adagrams {
+  // WAVE 1
+  static drawLetters() {
+    
+    let copyLetterPool = Object.assign({},letterPool)
+    let letters = Object.keys(letterPool);
+    let handOfLetters = [];
+
+  // draw 10 random letters 
+    while (handOfLetters.length < 10) {
+      let randomLetter = letters[Math.floor(Math.random() * letters.length)];
+        if (copyLetterPool[randomLetter] > 0) {
+            copyLetterPool[randomLetter] -= 1;
+            handOfLetters.push(randomLetter);
+          }
+        }
+      return handOfLetters; 
+    }
+
+  // WAVE 2
+  static usesAvailableLetters(input, lettersInHand) {
+      let inputArr = input.toUpperCase().split('');
+
+      for(let char of inputArr){
+        if (lettersInHand.includes(char)) {
+          let index = lettersInHand.indexOf(char);
+          lettersInHand.splice(index, 1);
+        } else {
+          return false; 
+        }
+      }
+      return true;
+    }
```

Comment:
This is over indented by two spaces.

I think the `static` keyword confused VS Code.